### ISSUE TITLE
[Expo 51 SDK] Cannot access 'colorKitUI' before initialization on Expo SDK Web

### ISSUE THREAD

**Issue by kennethstarkrl** [alabsi91/reanimated-color-picker#80]
### Description

When trying to use the color picker with Expo SDK 51 the following error occurs. This issue only occurs on Web. I can confirm it doesn't happen on Android, not sure about iOS.
```
Static Rendering Error (Node.js)
Cannot access 'colorKitUI' before initialization
Call Stack
factory
node_modules/reanimated-color-picker/lib/commonjs/colorKit/index.js
loadModuleImplementation
node_modules/metro-runtime/src/polyfills/require.js
guardedLoadModule
node_modules/metro-runtime/src/polyfills/require.js
require
node_modules/metro-runtime/src/polyfills/require.js
factory
node_modules/reanimated-color-picker/lib/commonjs/ColorPicker.js
loadModuleImplementation
node_modules/metro-runtime/src/polyfills/require.js
guardedLoadModule
node_modules/metro-runtime/src/polyfills/require.js
require
node_modules/metro-runtime/src/polyfills/require.js
factory
node_modules/reanimated-color-picker/lib/commonjs/index.js
loadModuleImplementation
node_modules/metro-runtime/src/polyfills/require.js
```

### Steps to reproduce

Create a bare Expo SDK 51 project and import reanimated-color-picker

### Reanimated Color Picker version

3.0.3

### React Native version

0.74.1

### React Native Reanimated version

3.10.1

### React Native Gesture Handler version

2.16.1

### Platforms

Web

### JavaScript runtime

Hermes

### Workflow

Expo bare workflow

### Architecture

Paper (Old Architecture)

### Build type

Debug mode

**Comment by kennethstarkrl** [comment]
I was able to fix it by doing the following

moved 
```js
exports.colorKitUI = colorKitUI;
```
 to the bottom of the file in `node_modules\reanimated-color-picker\lib\commonjs\colorKit\index.js`


moved
```js
exports.ConditionalRendering = ConditionalRendering;
exports.HSVA2HSLA_string = HSVA2HSLA_string;
exports.RenderNativeOnly = RenderNativeOnly;
exports.RenderWebOnly = RenderWebOnly;
exports.clamp = clamp;
exports.enableAndroidHardwareTextures = void 0;
exports.getStyle = getStyle;
exports.isWeb = exports.isRtl = void 0;
```
to the bottom of the file in `node_modules\reanimated-color-picker\lib\commonjs\utils.js`

Not sure how to change that in typescript otherwise I would have made a PR.

Thanks

**Comment by alabsi91** [comment]
I just published a new version to npm, `v3.0.4`, which includes a fix.
